lnrpc/walletrpc/walletkit.proto

```diff
@@ -64,6 +64,13 @@ service WalletKit {
     */
     rpc ListAccounts (ListAccountsRequest) returns (ListAccountsResponse);
 
+    /*
+    RequiredReserve returns the required reserve for a wallet along with its
+    balance.
```

Comment:
This could also use some more information/context. Feel free to use the same text I commented above.